core/src/test/java/software/amazon/awssdk/core/util/EC2MetadataUtilsServer.java

```diff
@@ -33,9 +33,9 @@
 
     private ServerSocket server;
 
-    public EC2MetadataUtilsServer(String address, int port)
+    public EC2MetadataUtilsServer(int port)
             throws UnknownHostException, IOException {
-        server = new ServerSocket(port, 1, InetAddress.getByName(address));
+        server = new ServerSocket(port, 1, InetAddress.getByName("localhost"));
```

Comment:
should we connect directly to `127.0.0.1` instead so there's no lookup?

Reply:
yup! updated

Reply:
Apprently `getLocalHost()` is actually kinda misleading and doesn't return the address of `localhost`: https://stackoverflow.com/a/18488982

Looking at it again, I think what you had originally is fine.

Reply:
yeah, actually, `InetAddress.getLoopbackAddress()` is what we want.

Reply:
👍 nice